### ISSUE TITLE
Running lighthouse (Chrome Audit) switches into mobile emulation automatically

### ISSUE THREAD

**Issue by danielrolfe** [GoogleChrome/lighthouse#3186]
Running either via the Chrome Dev Tools or the Chrome Extension causes the browser to render in mobile device emulation mode. This causes the site being tested to run as mobile and impacts the collected metrics. This is intermittent, occasionally I can make the audit run and render the site with a desktop user agent. I can reproduce this with multiple public sites such as https://www.royalcaribbean.com and https://www.carnival.com 

The device being emulated always shows as Nexus 5X.

**Comment by patrickhulce** [comment]
@danielrolfe this is intentional behavior and should happen every time. Lighthouse should always load your site with mobile emulation on in the extension and in DevTools. If you can render the site with a desktop user agent, repro steps would be much appreciated as this is a bug :)

You can follow the progress of making mobile emulation an option in #2838 or run from the CLI as mentioned in that thread.

**Comment by danielrolfe** [comment]
@patrickhulce Thank you, I assumed it would be the case as the problem was too obvious. However, I've haven't seen in the documentation, at least high level documentation that Lighthouse is a Mobile audit tool. So I must admit, I am somewhat confused by this and why Google would choose this as their primary Audit tool in Chrome? Any links with discussion or background on this would be appreciated, I previously searched for this information but to no avail.

**Comment by jalooc** [comment]
I'd also love to test my desktop version of web page using the Lighthouse auditing mechanisms - is this possible? I wasn't able to Google any info on that.

**Comment by buy** [comment]
echo what jalooc mentioned, is there a way to run desktop site? We have 2 different versions for desktop and mobile, with mobile view being the default one there's no way for us to run the audit for our desktop site. Previously it was possible by installing the [ADT chrome extension](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en), but now the audit entry has been claimed by lighthouse. Seems the only option we have is downgrade our Chrome?

**Comment by danielrolfe** [comment]
@jalooc @buy Honesty, I'm puzzled by why Google choose this tool. Desktop traffic is still pretty significant! Confused.

**Comment by mdr1384** [comment]
Also after the audit the device mode is in a weird state - the page is still showing in mobile mode but the DevTools device mode icon is not highlighted.  You have to turn on the device mode and then turn it off again for the page to revert back to desktop mode.

**Comment by banksean** [comment]
I too am super confused by this behavior. Why is desktop auditing specifically excluded? Seems like an anti-feature.

**Comment by patrickhulce** [comment]
FYI @banksean and others, the toggle for desktop/mobile landed in Chrome ~4 weeks ago and as always has been available on the CLI, use Chrome Canary or the CLI if you want this functionality today.

**Comment by aetcoff** [comment]
@patrickhulce I attended a talk at CSUN where they demoed the emulation mode for desktop, but I'm struggling to figure out how/where to select desktop as device mode. Can you share over a screenshot of where you can override the mobile mode?

**Comment by patrickhulce** [comment]
![image](https://user-images.githubusercontent.com/2301202/38380116-a87c43f4-38b7-11e8-95cd-a5a3aa54ccc7.png)

**Comment by aetcoff** [comment]
It looks like I don't have those same options and as far as I know, I'm at the latest version. 
![image](https://user-images.githubusercontent.com/7228610/38380229-face5e26-38b7-11e8-8afb-98a3cf65c8b9.png)

**Comment by patrickhulce** [comment]
@aetcoff you're probably using Chrome stable, features take ~12 weeks to make their way through all the channels. Use [Canary](https://www.google.com/chrome/browser/canary.html) if you want the latest features.

**Comment by aetcoff** [comment]
Got it! Thank you so much. :)

**Comment by banksean** [comment]
TL;DR:

npm install -g lighthouse
lighthouse --disable-device-emulation --disable-network-throttling --view <your url>

**Comment by ognik777** [comment]
I also have both a mobile and a desktop site.  On my Chrome (71.0...) - although there is an option for desktop (see snip) - it always runs as mobile.  i.e. selecting desktop still runs as mobile

![image](https://user-images.githubusercontent.com/43224653/51433494-35909380-1cb0-11e9-914e-4fef4f0249a8.png)

**Comment by Malvoz** [comment]
This is true for both the Chrome extension and the Audit panel in devtools.

**Comment by Hoten** [comment]
Fixed in Chrome 72 (see #6772), which releases tomorrow.

**Comment by david007co** [comment]
> 
> 
> Fixed in Chrome 72 (see #6772), which releases tomorrow.

It got Fixed as you mentioned in Chrome 72 (stable).
Thanks!

**Comment by quangtho2910** [comment]
I'm using the latest puppeteer with lighthouse and there is no way I can force the tool to audit the desktop site. Is there anyone else who has the same problem!

**Comment by patrickhulce** [comment]
@quangtho2910 see https://github.com/GoogleChrome/lighthouse/discussions/12058

**Comment by Raghavendra1178** [comment]
how to change configuration of lighthouse from mobile to website programmatically
